src/player.ts

```diff
@@ -38,6 +38,95 @@ export interface PlayerConfig {
    * ```
    */
   licenseKey?: string;
+  /**
+   * Configures playback behavior. A default `PlaybackConfig` is set initially.
+   */
+  playbackConfig?: PlaybackConfig;
+}
+
+/**
+ * Optionally PlaybackConfig Object used to configure a new `PlaybackConfig` instance.
+ */
+export interface PlaybackConfig {
+  /**
+   * Optionally isAutoplayEnabled: Specifies whether autoplay is enabled.
+   *
+   * Default is `false`.
+   *
+   * @example
+   * ```
+   * const player = new Player({
+   *   {
+   *     isAutoplayEnabled: true,
+   *   }
+   * })
+   * ```
+   */
```

Comment:
```suggestion
   * Whether the player starts playing automatically after loading a source or not. Default is `false`.
   *
   * @example
   * ```
   * const player = new Player({
   *   playbackConfig: {
   *     isAutoplayEnabled: true,
   *   },
   * });
   * ```
   */
```